### PR TITLE
[quest] ADDED: 'Sniper Training' Westfall Hunter Fake Rune Quest To QuestDB

### LINKED ISSUE #5452: [quest] 'Sniper Training' Westfall Hunter Rune Fake Quest Missing From QuestDB
## Description

The SoD Hunter fake rune quest 'Sniper Training' Westfall is not currently in the QuestDB. 

## Wowhead link

https://www.wowhead.com/classic/guide/season-of-discovery/hunter-class-runes-discovery-location
https://www.wowhead.com/classic/npc=210549/defias-scout
https://www.wowhead.com/classic/item=208777/rune-of-the-sniper
https://www.wowhead.com/classic/spell=416091/engrave-pants-sniper-training

## Screenshots

N/a

## Questie & WoW version

v9.2.14
WoW Classic SoD - v1.15.052409

### PR DESCRIPTION
**Pull request by Sommos**
## Issue references

Fixes #5452 
Linked To #5443 

## Proposed changes

- ADDED: 'Sniper Training' Westfall Hunter Fake Rune Quest is now in the QuestDB, and should be accessible to users.

### CODE REVIEW

File: Database/Corrections/sodQuestFixes.lua [hunk 1]
```diff
@@ -2502,6 +2502,18 @@ function SeasonOfDiscovery:LoadQuests()
...
         },
+        [90151] = {
+            [questKeys.name] = "Sniper Training",
+            [questKeys.startedBy] = {{210549}},
+            [questKeys.finishedBy] = nil,
+            [questKeys.requiredLevel] = 1,
+            [questKeys.questLevel] = 15,
+            [questKeys.requiredRaces] = raceIDs.NONE,
+            [questKeys.requiredClasses] = classIDs.HUNTER,
+            [questKeys.objectivesText] = {"Kill Defias Scouts until the rune drops."},
```
Comment (BreakBB) [inline Database/Corrections/sodQuestFixes.lua:2513]:
```suggestion
            [questKeys.objectivesText] = {"Spot a Defias Scouts with Hunter's Mark and defeat them until the rune drops."},
```

File: Database/Corrections/SeasonOfDiscovery.lua [hunk 2]
```diff
@@ -288,6 +289,7 @@ local questsToBlacklistBySoDPhase = {
         [90146] = true, -- Hiding Druid Mangle Mulgore for now as there are too many icons
         [90147] = true, -- Hiding Paladin Hand of Reckoning Westfall for now as there are too many icons
         [90148] = true, -- Hiding Paladin Exorcist Duskwood for now as there are too many icons
+        [90151] = true, -- Hiding Sniper Training Westfall for now as there are too many icons
```
Comment (BreakBB) [inline Database/Corrections/SeasonOfDiscovery.lua:292]: The spawns need to be cleaned up. The scout only spawns in two locations.

**Review by BreakBB — CHANGES_REQUESTED** [review]

**Comment by BreakBB** [comment]
It would also be nice to add this quest to this table, to mark it as Elite quest: https://github.com/Questie/Questie/blob/14b6939a8bb55e25a1def8cd2b8214bbcbf67233/Database/QuestieDB.lua#L56

File: Database/Corrections/sodQuestFixes.lua (hunk 1 above)
Comment (MyTechnoHunter) [reply Database/Corrections/sodQuestFixes.lua:2513]: No
Sentence malformed, reads wrong
"Find and mark Defias Scouts with Hunter's Mark, kill them while marked till the rune drops"

The above reads better

Comment (BreakBB) [reply Database/Corrections/sodQuestFixes.lua:2513]: Good point, thanks!

Comment (Sommos) [reply Database/Corrections/sodQuestFixes.lua]: Fixed in: https://github.com/Questie/Questie/pull/5453/commits/e1056afed1e95c1bed92e9912ea3adcbf6494aa6

File: Database/Corrections/SeasonOfDiscovery.lua (hunk 2 above)
Comment (Sommos) [reply Database/Corrections/SeasonOfDiscovery.lua]: Fixed in: https://github.com/Questie/Questie/pull/5453/commits/45922c2b1b46f2cda9fe9f61f059a42992a1bc99

**Pull request merged**